de/manual/ch_Finance-Quote.xml

```diff
@@ -253,222 +271,175 @@ SYNOPSIS
+  finanzpartner             fondsweb     fool         ftfunds      fundata
+  fundlibrary  goldmoney    greece       hu           hufund       hungary
+  hustock      iexcloud     india        indiamutual  known_currencies
+  morningstarau             morningstarch             morningstarjp
+  mstaruk      nasdaq       nseindia     nyse         nzx          onvista
+  oslobors     romania      seb_funds    six          tesouro_direto
+  tiaacref     tmx          tradeville   troweprice   troweprice_direct
+  tsp          ukfunds      unionfunds   usa          usfedbonds   yahoo_json
+  za
 </screen>
       </informalexample>
 
-      <para>Der erste Teil ist die Fehlermeldung über ein fehlendes &app-perl;-Modul Mozilla/CA.pm. Am Ende in
-        Klammern steht die normale Ausgabe der aktuell installierten &app-fq;-Version<footnote>
-        <para>Die neueste &app-fq; Version ist &app-fq-vers;.</para>
-        </footnote>
-        und eine Liste der verfügbaren Kursdaten-Quellen.
+      <para>Die neueste &app-fq; Version ist &app-fq-vers;.
       </para>
 
-      <tip>
-        <para>Wenn ein Fehler angezeigt wird, lesen Sie den nächsten Abschnitt <xref linkend="gnc-fq-update" />.
-          Ansonsten fahren Sie mit <xref linkend="gnc-fq-dump" /> fort.
-        </para>
-      </tip>
-    </sect2>
-
-    <sect2 id="gnc-fq-update">
-      <title>gnc-fq-update</title>
-
-      <abstract>
-        <para>Dieses Programm installiert oder aktualisiert das &app-fq; Softwaremodul und löst bei Bedarf
-          die Abhängigkeiten zu weiteren Softwarepaketen auf.
-          <note>
-            <para>Windows-Benutzer sind im Allgemeinen besser beraten, wenn sie das Update-Tool
-              &mc.winOS.inst-fq; ausführen.
-            </para>
-          </note>
-        </para>
-      </abstract>
-
-      <note>
-        <para>Dieses Programm erfordert für eine Installation unter &lin; oder &mac; Superuser-
-          oder Administratorrechte, aber nicht in &win;.
-        </para>
-      </note>
-
-      <para><userinput>gnc-fq-update</userinput> startet &app-perl;s Standardmodul
-        <acronym><ulink url="&url-cpan;">CPAN</ulink></acronym>
-        <footnote>
-          <simpara>Siehe <ulink url="&url-cpan;misc/cpan-faq.html">CPAN Frequently Asked Questions</ulink> für weitere
-            Informationen.
-          </simpara>
-        </footnote>
-        um die Installation vorzunehmen. Wenn Sie CPAN zum ersten Mal verwenden,
-        müssen Sie es konfigurieren, wobei Sie auf den meisten Systemen jedoch
-        die Standardvorgaben akzeptieren und die erste Frage <computeroutput>Are you ready for
-        manual configuration? [yes]</computeroutput> mit <userinput>no</userinput> beantworten können,
-        damit &app-fq; erfolgreich installiert wird.
+      <para>Wenn es ein Problem mit Ihrer Installation gibt, werden Sie darauf
+        hingeweisen. In diesem Fall fehlen zum Beispiel die &app-perl;-Module
+        Finance::Quote und JSON::Parse.
```

Comment:
Nein, nur JSON::Parse fehlt. Aber dadurch kann Finance::Quote nicht gebaut werden/starten.

Reply:
Doch, die Aussage ist hier richtig, weil der Befehl `gnucash-cli --quotes info` ggf. auch darauf hinweist, dass F::Q noch gar nicht installiert ist.

Reply:
Streng genommen müßte es heißen `FQ kann nicht gebaut werden, weil JSON::Parse nicht gefunden wurde.`

Reply:
ok, ich ergänze diesen Abschnitt mit erläuterndem Text um den Anwender in dieser Situation zu unterstützen.